src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,38 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Lotto {
+    LottoPrice price;
+    List<LottoNumbers> lottoNumbers;
```

Comment:
```suggestion
    private final LottoPrice price;
    private final List<LottoNumbers> lottoNumbers;
```
인스턴스 변수는 가능하면 private으로